cli/tools/registry/pm/outdated.rs

```diff
@@ -179,6 +180,31 @@ pub async fn outdated(
   let jsr_fetch_resolver =
     Arc::new(JsrFetchResolver::new(file_fetcher.clone()));
 
+  if !cli_options.start_dir.has_deno_json()
+    && !cli_options.start_dir.has_pkg_json()
+  {
+    bail!(
+      "No deno.json or package.json in {:?}.",
+      cli_options.initial_cwd(),
+    );
+  }
+
+  let lockfile_exists = match &cli_options.maybe_lockfile() {
+    Some(lockfile) => lockfile.filename.exists(),
+    None => false,
+  };
+
+  if !lockfile_exists {
+    bail!(
+      "{}",
+      color_print::cformat!(
+        "No lockfile in {:?}. Run <u>{}</> to generate one.",
+        cli_options.initial_cwd(),
+        "deno install"
+      )
+    );
+  }
```

Comment:
What happens when someone has `"lock": false`? I feel like this would break them?